web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('web.kanban_editable', function(require){
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
+
+    var _t = core._t;
+
+    var many2one_kanban = AbstractField.extend({
+        template: _t('Kanban_editable_temp'),
+        events: {
+            'click .selectM2MField': '_onSelectField',
+        },
+
+        init: function(){
+            this._super.apply(this, arguments);
+            if(!this.recordData[this.name])
+                this._updateField(this.recordData[this.name].id);
+            else
+                this._updateField(this.recordData[this.name].data.id);
+        },
+        _onSelectField: function(){
+            var self = this;
+            var val = this.$el.find('select[name="id"]').val();
+            var field_name = this.name;
+            var params = [[this.res_id],{
+                [field_name]: parseInt(val)
+            }];
+            this._rpc({
+                model: this.model,
+                method: 'write',
+                args: params
+            }).then(function(result){});
+            },
+        _updateField: function(curr_id) {
+            var self = this;
+            var model = this.field.relation
+            self._rpc({
+                model: model,
+                method: 'search_read',
+                fields: ['id', 'name'],
+                order: 'id'
+            }).then(function (data) {
+                // This is the first item empty for times when there is no data.
+                self.$el.find('.selectM2MField').append('<option' +
+                    ' value=""></option>');
+                for (var val of data) {
+                    self.$el.find('.selectM2MField').append(
+                        '<option value=' + val.id + '>' + val.name +
+                        '</option>');
```

Comment:
Why not change to:

```
'<option value=' + val.id + ((val.id == curr_id) ? 'selected' : '') +  '>'
```

instead of looking up the element again with jQuery like you do below